frontend/src/Editor/Components/Multiselect.jsx

```diff
@@ -125,6 +126,21 @@ export const Multiselect = function Multiselect({
     [selected, setSelected]
   );
 
+  const filterOptions = (options, filter) => {
+    if (!filter) return options;
+
+    setSearched(filter);
+
+    if (searched !== filter) {
+      fireEvent('onSearchTextChanged');
+      setExposedVariable('searchText', filter);
```

Comment:
The functionality is working as expected. However requesting a small change. We need call setExposedVariable before firing the event. So that we can make sure the variables are available for event handlers when it runs (ie: if fireEvent is made sync func)

Reply:
Hi @johnsoncherian,

Thanks for the feedback. You can check the updated code.